arangod/IResearch/IResearchAqlAnalyzer.h

```diff
@@ -84,6 +84,12 @@ class AqlAnalyzer final : public irs::analysis::analyzer{
     return "aql";
   }
 
+#ifdef TEST_F
```

Comment:
```suggestion
#ifdef ARANGODB_USE_GOOGLE_TESTS
```
Let's not assume test framework dependent macros is present